ros1_bridge/__init__.py

```diff
@@ -11,8 +11,33 @@
 # ROS 1 imports
 import genmsg
 import genmsg.msg_loader
+
+# import catkin_pkg and rospkg which are required by rosmsg
+# and likely only available for Python 2
+try:
+    import catkin_pkg
+except ImportError:
+    from importlib.machinery import SourceFileLoader
+    from subprocess import check_output
+    catkin_pkg_path = check_output(['python2', '-c', 'import catkin_pkg; print(catkin_pkg.__file__)'])
```

Comment:
Following the Python recommendations that is the only thing I could imagine. `python` might point to either 2 or 3.

Are the ROS 1 `catkin_pkg` and `rospkg` packages available when you use Python 3 on OS X?